src/Components/Input/Input.tsx

```diff
@@ -211,83 +160,18 @@ class Input extends React.PureComponent<Props, State> {
   };
 }
 
-type FieldProps = {
-  onChangeText(value: string): void;
-  inputBorderColor: string;
-  setInputRef(TextInput): void;
-  borderBottomWidth: number;
-  inputFontSize: number;
-  onSubmitEditing(e: NativeSyntheticEvent<TextInputSubmitEditingEventData>): void;
-  initialValue?: string;
-  secureTextEntry?: boolean;
-  textColor?: string;
-  nativeTextInputProps?: TextInputProps;
-  onFocus?(e: NativeSyntheticEvent<TextInputFocusEventData>): void;
-  onBlur?(e: NativeSyntheticEvent<TargetedEvent>): void;
-  keyboardType:
-    | "default"
-    | "email-address"
-    | "numeric"
-    | "phone-pad"
-    | "visible-password"
-    | "ascii-capable"
-    | "numbers-and-punctuation"
-    | "url"
-    | "number-pad"
-    | "name-phone-pad"
-    | "decimal-pad"
-    | "twitter"
-    | "web-search";
-};
-
-class Field extends React.Component<FieldProps> {
-  render() {
-    const {
-      initialValue,
-      secureTextEntry,
-      keyboardType,
-      textColor,
-      borderBottomWidth,
-      inputFontSize,
-      onFocus,
-      onBlur,
-      onChangeText,
-      inputBorderColor,
-      onSubmitEditing,
-      setInputRef,
-    } = this.props;
-    return (
-      <TextInput
-        {...this.props}
-        ref={setInputRef}
-        defaultValue={initialValue}
-        keyboardType={keyboardType}
-        style={[
-          styles.textInput,
-          { borderBottomColor: inputBorderColor, color: textColor, borderBottomWidth, fontSize: inputFontSize },
-        ]}
-        secureTextEntry={secureTextEntry}
-        onFocus={onFocus}
-        onBlur={onBlur}
-        onChangeText={onChangeText}
-        onSubmitEditing={onSubmitEditing}
-      />
-    );
-  }
-}
-
 const styles = StyleSheet.create({
   container: {
+    marginTop: 16,
+    flex: 1,
```

Comment:
Use more specific flex properties, so it will be easy to understand what exectly you want to change